packages/react-components/react-toast/src/state/types.ts

```diff
@@ -31,11 +34,11 @@ export interface CommonToastDetail {
   toasterId?: ToasterId;
 }
 
-export interface ShowToastEventDetail extends Partial<ToastOptions>, CommonToastDetail {
+export interface ShowToastEventDetail extends Partial<Omit<ToastOptions, 'dispatchedAt'>>, CommonToastDetail {
```

Comment:
nit: This `Omit<ToastOptions, 'dispatchedAt'>` seems to be repeating some times involving `dispatchToast`  (this might be too premature of me though, so don't mind it....)


```ts
export type DispatchToastOptions =  Partial<Omit<ToastOptions, 'dispatchedAt'>>
```

Reply:
I need to do a larger pass of typings later, I want to avoid creating a mass of typings at once until the API is more solidified